scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -49,7 +49,7 @@ def PEARL_startup(usern="matt", thiscycle='11_1'):
     # groupfile=pearl_file_dir + "test_cal_group_mods_11_1.cal"
     vabsorbfile = pearl_file_dir + "pearl_absorp_sphere_10mm_newinst_long.nxs"
     vanfile = pearl_file_dir + "van_spline_all_cycle_11_1.nxs"
-    attenfile = pearl_file_dir + "Attentuation\\PRL985_WC_HOYBIDE_NK_10MM_FF.OUT"
+    attenfile = "P:\Mantid\\Attentuation\\PRL985_WC_HOYBIDE_NK_10MM_FF.OUT"
```

Comment:
By the way it seems that the  mispelling `Attentuation` (note the t) has been in use and has propagated to all the scripts. Now all the files are in that place in \\britannic\Pearl\ it seams.